src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -1833,5 +1834,172 @@ public void Base_05()
             }
             EOF();
         }
+
+        [Theory]
+        [CombinatorialData]
+        public void DataProperty(
```

Comment:
>DataProperty [](start = 20, length = 12)

Consider also testing at the top level.

Reply:
> Consider also testing at the top level.

Including in script mode.

---
In reply to: [443080104](https://github.com/dotnet/roslyn/pull/44883#discussion_r443080104) [](ancestors = 443080104)